src/lib/Components/Inbox/Conversations/Message.tsx

```diff
@@ -125,7 +125,7 @@ export class Message extends React.Component<Props, any> {
           {this.renderAttachmentPreviews(message.attachments)}
 
           <BodyText disabled={!isSent}>
-            {message.raw_text.split("\n\nAbout")[0]}
+            {message.body.split("\n\nAbout")[0]}
```

Comment:
Yea, this is imperfect. We should move this, and other things like `VIEW FULL INQUIRY`, etc. upstream to Metaphysics (at least, or even Impulse/Radiation). Will do when we tackle more proper parsing/sanitization!

<img width="333" alt="screen shot 2017-07-27 at 6 22 34 pm" src="https://user-images.githubusercontent.com/1457859/28694599-aab65d10-72f8-11e7-8afd-37a6add9ea22.png">